src/api/collectionUnit/collectionUnit.router.ts

```diff
@@ -63,6 +64,16 @@ collectionUnitRouter.route("/critter/:id").get(
   })
 );
 
+collectionUnitRouter.route("/category/").get(
+  catchErrors(async (req: Request, res: Response) => {
+    const taxon_common = req.params.taxon_name_common;
```

Comment:
You could use a zod schema to parse the query params.

Reply:
You can add a nice error message if it fails

Reply:
I'm not sure if this needs to throw, I think it can just return empty array